src/calendarimportevent.h

```diff
@@ -38,9 +38,10 @@
 // KCalendarCore
 #include <KCalendarCore/Event>
 
+#include "calendardata.h"
 #include "calendarevent.h"
 
-class CalendarImportEvent : public QObject
+class CalendarImportEvent : public CalendarEvent
 {
     Q_OBJECT
     Q_PROPERTY(QString displayLabel READ displayLabel CONSTANT)
```

Comment:
Ponder if this set of redeclarations should be included. Feels interesting when subclass makes the properties constant but guess it's not necessarily wrong per se. Practical effect shouldn't be big when importing is rare and limited set of events.

Reply:
Yeh, I didn't want to change the previous implementation, but I agree that there is no point in declaring things constant here. I mean they are, but the performence delta in real life is neglectable I think. So I've removed the overloading declaration here.